packages/api2/src/periods/services/periods.service.ts

```diff
@@ -260,10 +225,12 @@ export class PeriodsService {
    * Return a PeriodDetails DTO for a given period
    *
    * @param {Period} period
-   * @returns {Promise<PeriodDetails>}
+   * @returns {Promise<PeriodDetailsDto>}
    * @throws {ServiceException} if period not found
    * */
-  findPeriodDetails = async (_id: Types.ObjectId) => {
+  findPeriodDetails = async (
```

Comment:
`PeriodDetailsDto` is not correctly serialized.

<img width="490" alt="image" src="https://user-images.githubusercontent.com/9698363/212646431-294e7441-9021-4e32-ad6b-0b91323c2027.png">